src/util/defaultVastaanottajat.ts

```diff
@@ -66,6 +69,22 @@ export default function defaultVastaanottajat(
         sahkoposti: "",
       })) || [];
   }
+  if (paatosTyyppi === PaatosTyyppi.JATKOPAATOS1 || paatosTyyppi === PaatosTyyppi.JATKOPAATOS2) {
+    if (ilmoituksenVastaanottajat?.maakunnat && ilmoituksenVastaanottajat.maakunnat.length > 0) {
+      maakunnat = ilmoituksenVastaanottajat.maakunnat.map((maakunta) => ({
+        id: maakunta.id,
+        sahkoposti: maakunta.sahkoposti,
+      }));
+    } else {
+      maakunnat =
+        projekti?.velho?.maakunnat?.map((id) => ({
+          id,
+          sahkoposti: "",
+        })) || [];
+    }
+  } else {
+    maakunnat = [];
+  }
```

Comment:
Eikös tässä pitäis tai kannattais aina asettaa maakunniksi projekti.velho.maakunnat listan maakunnat (ja etsiä niille maakunnille sähköpostiosoitteita ilmoituksenVastaanottajat.maakunnista? Nyt jos kertaalleen tallennetaan ilmoituksen vastaanottajat, ei sitä saa sieltä pois enää.

Niin tämä ongelmahan on tosiaan myös noissa kunnissa.

Reply:
Ei kai noita maakuntia/kuntia enää projektin aikana säädetä. Kyllähän ne pitäs olla jo tässä vaiheessa oikein Velhossa ja synkattu VLS:ään.

Reply:
Lisäksi tuo aiheuttaisi ongelman että käyttäjän pitäisi tajuta tallentaa projekti uudelleen kun maakunta poistettu Velhosta ja se synkattu VLS:ään. En nyt lähtisi tätä yhtään monimutkaistamaan, kun ei tämä käytännössä ole ongelma.

Reply:
Eihän tässä tosiaan maakuntien kohdalla ei ole ainakaan iso ongelma. Mutten myöskään nää tuota ehdotustani oikeastaan monimutkaistavana asiana.

Ongelmatilanne on todennäköisin migroiduilla projekteilla:
1. Projekti on migroitu jatkopäätösvaiheeseen
2. käyttäjä kerkeää kertaalleen tallentaa jatkopäätössivun niin, että siinä on "vääriä" kuntia ja maakuntia.
3. käyttäjä ei voi saada näitä kuntia ja maakuntia pois tai uusia kuntia tilalle kuulutukselle

Jos lomake on muokkattavana, se joka tapauksessa kuitenkin tulee tallennetuksi ennen kuin sitä on viety hyväksyttäväksi. Eli käyttäjän ei oikeastaan tarvitse tajuta muistaa tallentaa lomaketta, koska se tallennus tulee tapahtumaan.